pom.xml

```
@@ -66,8 +66,8 @@
 
   <!-- Bring some sanity to version numbering... -->
   <properties>
-    <revision>1.0.8</revision>
-    <changelist>-SNAPSHOT</changelist>
+    <revision>0</revision>
```

```suggestion
    <revision>1</revision>
```
Using any number lower than 1, starts cutting releases considered older than 1.0.8